docs/source/package_reference/file_download.mdx

```diff
@@ -4,8 +4,6 @@
 
 [[autodoc]] huggingface_hub.snapshot_download
 
-[[autodoc]] huggingface_hub.cached_download
```

Comment:
Why are we removing this?

Reply:
It's deprecated so I wouldn't promote the method in the package reference anymore.

Do you feel strongly otherwise?